packages/map-template/src/hooks/useMapBoundsDeterminer.js

```diff
@@ -165,82 +158,15 @@ export default useMapBoundsDeterminer;
  * @param {number} pitch
  * @param {number} bearing
  */
-function goToGeometry(mapType, geometry, mapsIndoorsInstance, paddingBottom, paddingLeft, zoomLevel, pitch, bearing) {
```

Comment:
Why has the mapType been removed here?

Reply:
The information is no longer needed, since it's the SDK that takes care of knowing which map provider to send instructions to.